config/nginx.conf.erb

```diff
@@ -0,0 +1,77 @@
+daemon off;
+# Heroku dynos have at least 4 cores.
+worker_processes <%= ENV['NGINX_WORKERS'] || 4 %>;
+
+events {
+  use epoll;
+  accept_mutex on;
+  worker_connections <%= ENV['NGINX_WORKER_CONNECTIONS'] || 1024 %>;
+}
+
+http {
+  gzip on;
+  gzip_comp_level 2;
+  gzip_min_length 512;
+
+  server_tokens off;
+
+  log_format l2met 'measure#nginx.service=$request_time request_id=$http_x_request_id';
+  access_log <%= ENV['NGINX_ACCESS_LOG_PATH'] || 'logs/nginx/access.log' %> l2met;
+  error_log <%= ENV['NGINX_ERROR_LOG_PATH'] || 'logs/nginx/error.log' %>;
+
+
+  include mime.types;
+  default_type application/octet-stream;
+  sendfile on;
+
+  # Must read the body in 5 seconds.
+  client_body_timeout <%= ENV['NGINX_CLIENT_BODY_TIMEOUT'] || 5 %>;
+
+  server {
+    listen <%= ENV["PORT"] %>;
+    server_name codefund.io;
+    keepalive_timeout 5;
+    client_max_body_size <%= ENV['NGINX_CLIENT_MAX_BODY_SIZE'] || 1 %>M;
+
+    location ~ \/scripts\/.+\/embed\.js {
+      root /app/public;
+    }
+
+    location ~ \/api\/v1\/impression\/.+ {
+      root /app/public;
+    }
+
+    location ~ \/t\/s\/.+\/details {
+      root /app/public;
+    }
+
+    location ~ \/properties\/.+\/funder\..+ {
+      root /app/public;
+    }
+
+    location ~ \/impressions\/display\/.+\.gif {
+      root /app/public;
+    }
+
+    location ~ \/impressions\/.+\/uplift {
+      root /app/public;
+    }
+
+    location ~ \/impressions\/.+\/click {
+      root /app/public;
+    }
+
+    location / {
+      proxy_pass https://codefundwww.wpengine.com;
```

Comment:
Can we make this https://wp.codefund.io ? That will send the traffic through cloudflare, but it will also give us more control and better metrics.